cmd/namespace.go

```diff
@@ -37,7 +37,8 @@ func createNamespace(cmd *cobra.Command, args []string) error {
 
 	response, err := api.CreateNamespace(ctx, Logger, args[0], args[2], strings.ToUpper(args[1]))
 
-	if err != nil {
+	// Only fall back to native graphql errors if there are no in-band errors.
+	if err != nil && response == nil && len(response.Errors) == 0 {
```

Comment:
```
response == nil && len(response.Errors) == 0
```
Should this be `||`?

Reply:
`&&` should be correct.

I'm basically shorting out that statement so we don't trying and access `.Errors` on a `nil` `response`, `nil.Errors` is not a good thing :)

Reply:
:facepalm: yep.. should be `!=`